CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -103,11 +104,17 @@ Describe "Single compute node protocol tests with utils" {
             -Name $NetworkName `
             -Subnet "$( $Subnet.IpPrefix )/$( $Subnet.IpPrefixLen )"
 
-        Write-Host "Creating containers"
-        $Container1ID, $Container2ID = Invoke-Command -Session $Session -ScriptBlock {
+        Write-Host "Creating container 1"
+        $Cmd1 = Invoke-NativeCommand -Session $Session -CaptureOutput {
```

Comment:
[Refinement] Do we need two variables Cmd1 and Cmd2? Maybe we could use only one variable like  $Command?

Reply:
I don't really like reusing vars